contracts/DLCManager.sol

```diff
@@ -258,6 +266,41 @@ contract DLCManager is
         return false;
     }
 
+    /**
+     * @notice  Checks mint eligibility.
+     * @dev     Checks if the amount is non-zero.
+     * @dev     If PoR is disabled, returns true.
+     * @dev     If PoR is enabled, checks if the new total value minted is within bounds.
+     * @dev     If the PoR check fails, reverts with an error.
+     * @param   amount  dlcBTC to mint.
+     * @param   currentTotalMinted  total minted value in all vaults.
+     * @return  bool  whether a call to _mint should happen.
+     */
+    function _checkMint(
+        uint256 amount,
+        uint256 currentTotalMinted
+    ) internal view returns (bool) {
+        if (amount == 0) {
+            return false;
+        }
+
+        if (!porEnabled) {
+            return true;
+        }
+
+        (, int256 latestAnswer, , , ) = dlcBTCPoRFeed.latestRoundData();
```

Comment:
does this have to be called 'latestAnswer' ?  it's not a very meaningful name for the function